<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test</class>
 <widget class="QWidget" name="test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>test</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;データタイプ（日経平均は「＾N225」、企業は証券コードに「.JP」を付けたコード）&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="buddy">
    <cstring>dataType</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;データソース（日経平均の場合は「yahoo」、企業の場合は「stooq」）&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="buddy">
    <cstring>dataSource</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;データ取得期間（日数で入力してください。）&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="buddy">
    <cstring>duration</cstring>
   </property>
  </widget>
  <widget class="QSpinBox" name="duration">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>50</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="value">
    <number>365</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="method">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>平均算出方法</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="sma">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>単純移動平均</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="wma">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>加重移動平均</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ewm">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>指数平滑移動平均</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>551</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;日別終値変動グラフ 詳細設定&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="graphButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Golden / Dead
Cross</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="dataType">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>^N225</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="dataSource">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>yahoo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="candleButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>230</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>CandleChart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getDataButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>getData</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>dataType</tabstop>
  <tabstop>dataSource</tabstop>
  <tabstop>duration</tabstop>
  <tabstop>sma</tabstop>
  <tabstop>wma</tabstop>
  <tabstop>ewm</tabstop>
  <tabstop>graphButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>checkType()</slot>
 </slots>
</ui>
